drivers/adc/ad7799/ad7799.c

```diff
@@ -235,17 +235,18 @@ int32_t ad7799_get_channel(struct ad7799_dev *device, uint8_t ch,
 }
 
 /**
- * @brief Read data in mV from specific ADC channel.
+ * @brief Read scaled data from specific ADC channel.
  * @param device - The device structure.
  * @param ch - The ADC channel.
- * @param data_mv - The content of the data converted in mV.
+ * @param data_scaled - The content of the scaled data
  * @return SUCCESS in case of success, negative error code otherwise.
  */
-int32_t ad7799_read_channel_mv(struct ad7799_dev *device, uint8_t ch,
-			       int32_t *data_mv)
+int32_t ad7799_read_channel_scaled(struct ad7799_dev *device, uint8_t ch,
```

Comment:
nitpick: I'am not sure if scaled shoud be added to the name. I would go for just removing mv and just specify in the comment the format of the expected result.